api.py

```diff
@@ -870,6 +870,35 @@ def create_api_key(**kwargs):
     return json.loads(response.content)['apikey']
 
 
+def delete_api_key(key, **kwargs):
+    """
+    Delete an API key from this user's account.
+
+    Send an HTTP POST request to delete the given API key.  The key in question may not be used for authentication of the request.
```

Comment:
Is the key destroyed, or does Conduce hang on to it?

If it destroyed, we should mention that here.  Otherwise, lets update the language to say:

>The key in question may no longer be used for authentication of the request.

Reply:
This is one of those methods that actually deletes something.  So when you call this method, the key is lost forever and cannot be recovered or restored.